### DISCUSSION TITLE
RHVoise cli (without DBus and services)

### DISCUSSION THREAD

**Discussion by QuAzI** [RHVoice/RHVoice#615] [Q&A]
I need to read text and save it into file. But from WSL it is impossible because
```
Unable to connect to the bus
```
and
```
Failed to connect to Speech Dispatcher:
Error: Can't connect to unix socket /home/quazi/.cache/speech-dispatcher/speechd.sock: No such file or directory. Autospawn: Autospawn failed. Speech Dispatcher refused to start with error code, stating this as a reason:
```

So there any way to use it just as CLI?
Also it is strange when main page of project doesn't contain information about setup and how to start it and WIKI is empty.
And no way to do `scons uninstall`

**Comment by beqabeqa473** [comment]
you can use RHVoice-test

On 8/10/22, Ruslan Yakauleu ***@***.***> wrote:
> I need to read text and save it into file. But from WSL it is impossible
> because
> ```
> Unable to connect to the bus
> ```
> and
> ```
> Failed to connect to Speech Dispatcher:
> Error: Can't connect to unix socket
> /home/quazi/.cache/speech-dispatcher/speechd.sock: No such file or
> directory. Autospawn: Autospawn failed. Speech Dispatcher refused to start
> with error code, stating this as a reason:
> ```
>
> So there any way to use it just as CLI?
> Also it is strange when main page of project doesn't contain information
> about setup and how to start it and WIKI is empty.
> And no way to do `scons uninstall`
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/RHVoice/RHVoice/discussions/615
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili
